website/content/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -7,7 +7,7 @@ description: Configuring Ingress Gateways on Kubernetes
 
 # Ingress Gateways on Kubernetes
 
--> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+-> 1.9.0+: This feature is available in Consul versions 1.9.0 and higher
```

Comment:
Bumped to 1.9.0 because we're using the `ServiceIntentions` CRD which requires consul 1.9.0